### PR TITLE
Update/button component

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55 

I made some minor changes to the Button component.  Can you please take a look and let me know if anything needs to be fixed?  

Thanks.

**Review by MaksymTaratuta55 — COMMENTED** [review]
@thecodeclimber, could you please clarify why you want to separate the button theme constant?
I think we should store all components styles in the one constant `theme`. But we can collect it from different files:

**src/themes/components/button.js**
```
const buttonTheme = {....};
export default buttonTheme;
```

**src/themes/components/typography.js**
```
const typographyTheme = {....};
export default typographyTheme;
```

**src/themes/theme.js**
```
import buttonTheme from './componrnts/...";
import typographyTheme from './componrnts/...";

const theme = {
     typography: typographyTheme,
     button: buttonTheme
};
```

And then use it as we used:
**src/components/Button.jsx**
```
const Button = styled.button(
  ({ className, styles: { block, color, border, shape, size } = {} }) => [
    theme.button.default.base,
    theme.button.default.color,
    theme.button.default.hover
]);
```

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55 

I just want to separate them into different files because I find it is too much if we put everything in one file.  But in terms of whether we use one constant or not -- can I ask what difference it makes? 

On a separate note, I just would like to clarify that I do not want to combine all the props into one `styles` prop.   I.e., rather then this:

```
const Button = styled.button(
  ({ className, styles: { block, color, border, shape, size } = {} }) => [
   ...
]);
```

I would like this:

```
const Button = styled.button(
  ({ className, block, color, border, shape, size }) => [
   ...
]);
```

**Comment by MaksymTaratuta55** [comment]
> I just want to separate them into different files because I find it is too much if we put everything in one file. But in terms of whether we use one constant or not -- can I ask what difference it makes?

I think as a result using one constant `theme` is a better logical way and then at any moment, you can add `darkTheme` or something like this and easily use it.

**Comment by MaksymTaratuta55** [comment]
> I would like this:
```
const Button = styled.button(
  ({ className, block, color, border, shape, size }) => [
   ...
]);
```

Sure you can use it so. I just wanted to separate style props in one common property. But it's ok if you don't want

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55 

Okay, do you mind making the changes?

**Comment by MaksymTaratuta55** [comment]
@thecodeclimber, no, I don't mind these changes

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55  Great -- thanks :)

**Comment by thecodeclimber** [comment]
@MaksymTaratuta55 

Please let me know when you have made the changes.

File: package.json [hunk 1]
```diff
@@ -28,7 +28,6 @@
     "postcss-preset-env": "^6.7.0",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-icons": "^4.1.0",
     "styled-components": "^5.2.1",
```
Comment (thecodeclimber) [inline package.json]: Please remove `styled-components`.  We are using `emotion`.

**Review by ghost — COMMENTED** [review]
@MaksymTaratuta55 

Are we using `styled-components` or `emotion`?  

Also, is there anything else we need to do to set up twin.macro:  https://github.com/ben-rogerson/twin.macro#get-started

**Comment by MaksymTaratuta55** [comment]
@thecodeclimber, updated

**Pull request merged**